README.md

```diff
@@ -17,4 +17,43 @@ As a user, I would like to customize the appearance or functionality of my featu
 
 ## What's next?
 So far, we have updated our output type to modify our head and footer on the page, created a basic layout to provide semantic html to our page, and began creating and modifying features using custom fields from
-Fusion's component context. Let's practice with an exercise - creating an alert bar! Switch to `exercise-01` to see one example implementation.
\ No newline at end of file
+Fusion's component context. Let's practice with an exercise - creating an alert bar! Switch to `exercise-01` to see one example implementation.
+
+If you are continuing this training from the last user story, then keep working on your current branch USERNAME-fusion-training.
+
+If you are starting this training from the seventh user story (task Fusion-07), then follow the steps under "What's next?" on the README file in the (https://github.com/wapopartners/Fusion-Training-User-Stories/blob/fusion-01/README.md)[fusion-01] branch - but checkout from fusion-06 instead.
+
+## New Task: User Story 07
+As a user, I would like to directly customize my feature using inline editing for a streamlined experience.
+
+## Guide
+Building on the alert bar feature from `exercise-01` we provide a clearer and easier way for editors to configure custom fields through inline editing as documented [here](https://redirector.arcpublishing.com/alc/arc-products/pagebuilder/fusion/documentation/recipes/adding-custom-fields.md#inline-editing)
+
+1. Complete the user story from branch `exercise-01`. ```git checkout exercise-01```
```

Comment:
instead of instructing them to check out from a new branch, how about we ask them to just download the feature from [exercise-01](https://github.com/wapopartners/Fusion-Training-User-Stories/blob/exercise-01/components/features/global/alert-bar/default.jsx)

That way, they can keep working off their same branch as we instructed them above: "...then keep working on your current branch USERNAME-fusion-training."